phoenix-core/src/main/java/org/apache/phoenix/schema/tool/SchemaExtractionProcessor.java

```diff
@@ -135,8 +135,8 @@ private String getIndexedColumnsString(PTable indexPTable, PTable dataPTable, St
         StringBuilder indexedColumnsBuilder = new StringBuilder();
 
         for (PColumn indexedColumn : indexPK) {
-            String indexColumn = extractIndexColumn(indexedColumn.getName().getString(), defaultCF);
-            if(indexColumn.equalsIgnoreCase(MetaDataUtil.VIEW_INDEX_ID_COLUMN_NAME)) {
+            String indexColumn = extractIndexColumn(indexedColumn.getExpressionStr(), defaultCF);
```

Comment:
so indexedColumn.getExpressionStr() will return the colum name for "simple" indexes ?

Reply:
We still parse it but indexedColumn.getExpressionStr() is better to parse.
indexedColumn.getName().getString() returnes 'QUOTED' for quoted string and ('number' * 'currency') for lowercase columns so we would not be able to differentiate them.
On the other hand  indexedColumn.getExpressionStr() returns these columns as 'QUOTED' and ("number" * "currency")